stablehlo/tests/verify_reduce_window.mlir

```diff
@@ -262,6 +268,27 @@ func.func @reduce_window_invalid_attributes(%arg0: tensor<4x2xf32>,
 
 // -----
 
+func.func @reduce_window_invalid_attributes(%arg0: tensor<4x2xf32>, %arg1: tensor<4x2xi32>,
+                    %init0: tensor<f32>, %init1: tensor<i32>) ->
+                      (tensor<2x2xf32>, tensor<2x2xi32>) {
+  // expected-error @+1 {{expects the shape of attribute to be 1-D}}
```

Comment:
Of what attribute?

Reply:
To address this, I add arg "StringRef attrName" to the signature of `onvertDenseIntAttr` so that we have a more concrete error message.